### DISCUSSION TITLE
migrate in Prisma show ERROR :

### DISCUSSION THREAD

**Discussion by Susanta-0007** [prisma/prisma#22834] [Q&A] [answered]
### Bug description

Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "aws-0-ap-south-1.pooler.supabase.com:6543"

Error: Schema engine error:
ERROR: prepared statement "s0" does not exist

### How to reproduce

Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "aws-0-ap-south-1.pooler.supabase.com:6543"

Error: Schema engine error:
ERROR: prepared statement "s0" does not exist

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
Did you see Supabase's documentation on how to use it with Prisma, and the mention of `pgbouncer=true` in there?
https://supabase.com/partners/integrations/prisma

**Reply by Susanta-0007** [reply, marked as answer]
![Screenshot 2024-01-27 133216](https://github.com/prisma/prisma/assets/89796600/8e9c9863-75a4-4168-b96a-666232051986)
---> Show the error .. How to resolve this

**Comment by jatingodnani** [comment]
@janpio how to resolve this
